R/tt_pos_and_access.R

```diff
@@ -306,10 +306,11 @@ setMethod(
   while (length(cur_path > 0)) {
     kids <- tree_children(cur_tbl)
     curname <- cur_path[1]
+    kids_names <- sapply(kids, obj_name)
```

Comment:
from the line 309 to the line 333, this is a complex scenario, thanks for considering all cases, @Melkiades , can you please add explicit test case for each scenario path

Reply:
Yes! I would move it to another issue as this needs a bit of time to understand when this can be used and extend behavior. For the moment, it is only a safe-net and I think we need this PR in for as_result_df first

Reply:
Opened a new issue regarding this